tests/dummy/app/states/todos.js

```diff
@@ -1,6 +1,8 @@
 import BufferedProxy from 'ember-buffered-proxy/proxy';
 
-BufferedProxy.reopenClass({
+const TodosProxy = BufferedProxy.extend();
+
+TodosProxy.reopenClass({
   initialState() {
     return {
       content: this.get('todo')
```

Comment:
@stefanpenner initialState gets invoked like so: https://github.com/stefanpenner/ember-state-services/blob/master/addon/state-for.js#L72-L74. This sets the context of this to the consuming component/controller/whatever. It might be better/more straight forward to pass it in as an argument. Thoughts?